omod/src/main/webapp/resources/htmlforms/sierra_leone/section-exam.xml

```diff
@@ -242,26 +242,34 @@
             </h3>
 
             <div class="two-columns exam">
-                <p>
+                <!--<p>
                     <obs class="normalObs" conceptId="PIH:GENERAL EXAM FINDINGS"
                          answerConceptId="PIH:NORMAL" style="checkbox"
                          />
-                </p>
+                </p>-->
 
                 <!-- ToDo:  UHM-3529 Add toggle for normal/abnormal symptoms
                 <div id="abnGenExam"> -->
+                
                 <p>
-                    <obs class="abnormalObs" conceptId="PIH:GENERAL EXAM FINDINGS" answerConceptId="PIH:WEIGHT LOSS" style="checkbox"/>
```

Comment:
By removing the class of normalObs and abnormalObs, the checkboxes probably won't toggle as designed.  When "normal" is selected, it unsets the checkboxes for the abnormal symptoms/findings.  When an "abnormal" sign/symptom checkbox is selected (ie. weight loss), "normal" checkbox is unchecked.  Is that the behavior that you want @lebbiesahr ?

Reply:
No, that wasn't my intention will review and push again. Thanks.